provider/clusterrosaclassic/cluster_rosa_classic_resource.go

```diff
@@ -393,22 +395,29 @@ func (r *ClusterRosaClassicResource) Schema(ctx context.Context, req resource.Sc
 					"upgrade to OpenShift 4.12.z from 4.11 or before).",
 				Optional: true,
 			},
+			"create_admin_user": schema.BoolAttribute{
+				Description: "Indicates if create cluster admin user",
```

Comment:
Please elaborate on all the optional usages of that attribute with (and without) the `admin_credentials` attribute

Reply:
added more details in the Description